libbuteosyncfw/plugin/MultiSyncResultModel.cpp

```diff
@@ -0,0 +1,159 @@
+/*
+ * This file is part of buteo-syncfw package
+ *
+ * Copyright (C) 2020-2021 Damien Caliste.
+ *
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA
+ *
+ */
+
+#include "MultiSyncResultModel.h"
+
+#include <QDebug>
+
+using namespace Buteo;
+
+bool ProfileEntry::operator<(const struct ProfileEntry &other)
+{
+    return label < other.label;
+}
+
+MultiSyncResultModel::MultiSyncResultModel(QObject *parent)
+    : SyncResultModelBase(parent)
+    , mSortOption(MultiSyncResultModel::ByDate)
+{
+    beginInsertRows(QModelIndex(), 0, 0);
```

Comment:
Suppose this should know the number of added rows, but then again in the ctor there shouldn't be anyone able to track the changes.

Reply:
Do you think then that I can completely remove the `beginInsert` and `endInsert` calls from the constructor ? If they are just for signal emission, I agree they are useless. But I was wondering if you need them for the model to actually do some internal stuff after being populated.

Reply:
Yea think those can be removed from here.

Reply:
Ok, begin/end InsertRows() removed.